### DISCUSSION TITLE
[Bug]: Monitoring is broken, again.

### DISCUSSION THREAD

**Discussion by haywardgg** [GameServerManagers/LinuxGSM#4665] [General]
### User story

As a user I'd like this to work.

### Game

7 Days to Die

### Linux distro

Ubuntu 22.04

### Command

command: monitor

### Further information

This is still an issue: https://github.com/GameServerManagers/LinuxGSM/pull/3764

I have to edit the config and change the monitoring type from 2 to 5.

**Comment by dgibbs64** [comment]
I have confirmed that protocol-valve does worth with 7dtd and I have it working on my own test server. I am unsure exactly what is causing the issue in your setup specifically. Transferring to a discussion as I have confirmed query is working
